pages/osx/stern.md

```diff
@@ -0,0 +1,20 @@
+# Stern
+
+> Tail Logs form multiple pods and multiple containers on Kubernetes.
+> Reference: <https://github.com/wercker/stern>.
+
+- Tail logs of container "nginx" inside pod "nginx-pod" from namespace "production":
+
```

Comment:
Shall we replace the specific container name and pod name with a placeholder?

Reply:
done, thanks for suggesting.